src/main/app/auth/user/oidc.ts

```diff
@@ -38,20 +38,18 @@ export const getUserDetails = async (
 };
 
 export const getSystemUser = async (): Promise<UserDetails> => {
-  const id: string = config.get('services.idam.clientID');
+  //const id: string = config.get('services.idam.clientID');
   const secret: string = config.get('services.idam.clientSecret');
   const tokenUrl: string = config.get('services.idam.tokenURL');
 
   const systemUsername: string = config.get('services.idam.systemUsername');
   const systemPassword: string = config.get('services.idam.systemPassword');
 
   const headers = { Accept: 'application/json', 'Content-Type': 'application/x-www-form-urlencoded' };
-  const data = `grant_type=password&username=${systemUsername}&password=${systemPassword}&client_id=${id}
-                &client_secret=${secret}&scope=openid%20profile%20roles%20openid%20roles%20profile`;
+  const data = `grant_type=password&username=${systemUsername}&password=${systemPassword}&client_id=prl-cos-api&client_secret=${secret}&scope=openid%20profile%20roles%20openid%20roles%20profile`;
```

Comment:
Can we not use the 'services.idam.clientID' constant so that we just need to change the value of it at 1 place when we will have the correct key available?